tests/createWcsWithSip.py

```diff
@@ -79,6 +78,7 @@ def testBigXy0(self):
             src.set(yKey, src.get(yKey) - 500 + y0)
         bbox = afwGeom.Box2I(afwGeom.Point2I(x0, y0), afwGeom.Extent2I(1000, 1000))
         res = self.astrom.determineWcs2(cat, bbox=bbox)
+        self.assertTrue(res.sipWcs is not None, "Failed to fit SIP terms")
```

Comment:
The message includes the code that was False, so the message is sufficiently clear. With conditionals, e.g. a > b you don't see the individual values. I'm willing to do it, but I don't think it's very useful and those ugly long method names start to get to me after awhile. I didn't even know that one existed. It feels like clutter to me.